module-batch/src/main/java/com/example/modulebatch/user/db/entity/UserStatistics.java

```diff
@@ -0,0 +1,32 @@
+package com.example.modulebatch.user.db.entity;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.Getter;
+
+@Entity
+@Getter
+public class UserStatistics {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long user_stat_id;
+
+	@Column(name = "statistics_at", unique = true)
+	private LocalDateTime statisticsAt;
+
+	@Column(name = "registered_count")
+	private Integer registeredCount;
+
+	public static UserStatistics create(int registeredCount) {
+		UserStatistics statistics = new UserStatistics();
+		statistics.registeredCount = registeredCount;
+		statistics.statisticsAt = LocalDateTime.now().minusDays(1);
+		return statistics;
+	}
```

Comment:
주간 통계를 만들기 위해 statisticsAt을 파라미터로 받아오는 방식으로 수정하면 어떨까요?
```suggestion
        public static UserStatistics create(int registeredCount, LocalDateTime statisticsAt) {
                UserStatistics statistics = new UserStatistics();
                statistics.registeredCount = registeredCount;
                statistics.statisticsAt = statisticsAt;
                return statistics;
        }
```

Reply:
statisticsAt 은 해당 통계가 작성된 날짜이기 때문에 외부에서 받아오기 보다는 Statistics 엔티티가 create메서드 내부에서 직접 생성하는 책임을 갖는게 맞다고 생각합니다

Reply:
groupby로 일간에서 주간 조회수 구할거면 외부에서 받아올 필요가 없을 것 같네요, 이해했습니다.